src/specter/Specter.php

```diff
@@ -60,7 +60,6 @@ public function onCommand(CommandSender $sender, Command $command, string $label
                     } else {
                         return false;
                     }
-                    break;
```

Comment:
hmm. not too sure about this one. i once heard that it is only safe to remove them if the "case" got parenthesis around it

Reply:
It's unreachable because all branches in this block have return statements in them.